ci_build.sh

```diff
@@ -4,17 +4,9 @@
 #  THIS FILE IS 100% GENERATED BY ZPROJECT; DO NOT EDIT EXCEPT EXPERIMENTALLY  #
 #  READ THE ZPROJECT/README.MD FOR INFORMATION ABOUT MAKING PERMANENT CHANGES. #
 ################################################################################
-# NOTE: This script was customized (PATH, added commented-away quicktest lines,#
```

Comment:
Customizations below are lost. Some of these might be responsible for travis fails of this current mix of legacy+generated codebase.